build.gradle

```diff
@@ -22,6 +22,7 @@ allprojects {
 
 checkstyle {
     maxWarnings = 0
+    ignoreFailures = false
```

Comment:
Because we have severity set up as warning and not errors, will this setting be of any help?